### PR TITLE
Shuttle Console Radar improvements

### PR DESCRIPTION
**Pull request by GreaseMonk**
## About the PR

Theres big changes to the map from this PR https://github.com/space-wizards/space-station-14/pull/24430
but it has a long time to go until it gets here.

- Added blips / triangles over ships and points towards ships when its outside radar circle
- Reduced font size
- dont show decimals for distance if >50m
- 1295m -> 1.3km  shows km's now instead of m to save some screen space
- added hover mouse over

Old:
image

new:
image

## Why / Balance

figured i can make some QOL improvements to current radar

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

tweak: Radar QOL improvements

### CODE REVIEW

**Comment by exincore** [comment]
The isosceles triangles at the rim of the radar view are rotated 120 degrees clockwise.

**Comment by exincore** [comment]
The mouse position is different on different UI scales

**Comment by exincore** [comment]
I did some more testing and it seems less to do with scaling and more to do with the offset?

File: Content.Client/Shuttles/UI/RadarControl.cs [hunk 1]
```diff
@@ -23,10 +25,15 @@ public sealed class RadarControl : MapGridControl
...
     [Dependency] private readonly IMapManager _mapManager = default!;
-    private SharedTransformSystem _transform;
+    [Dependency] private readonly IResourceCache _resourceCache;
+    [Dependency] private readonly IUserInterfaceManager _uiManager = default!;
+    private readonly SharedTransformSystem _transform;
 
     private const float GridLinesDistance = 32f;
 
+    private const int RadarBlipSize = 15;
+    private const int RadarFontSize = 10;
```
Comment (Cheackraze) [inline Content.Client/Shuttles/UI/RadarControl.cs:35]: possibly move these to the component in the future

**Review by Cheackraze — CHANGES_REQUESTED** [review]
looks good to me, though this is above me in many places. I only ask that any changes to upstream code have comments added in-line or block style denoting its a frontier change, along with a 1 sentence overview of the change if possible. This is so that maintainers can more easily spot and recognize necessary changes to the code.

File: Content.Client/Shuttles/UI/RadarControl.cs (hunk 1 above)
Comment (Cheackraze) [reply Content.Client/Shuttles/UI/RadarControl.cs:35]: not necessary for this PR though

**Pull request merged**